### PR TITLE
[OAP-1523][oap-native-sql]clean up metric

### LINKED ISSUE #1523: [oap-native-sql] clean up metrics
each operator should provide a total time

- [x] batchscan
- [x] condproject
- [x] hashagg
- [x] hashjoin
- [x] sort
- [x] RowToColumnar
- [ ] exchange

### PR DESCRIPTION
**Pull request by zhouyuan**
this patch cleans up metrics by adding total time

fixes #1523

### CODE REVIEW

File: oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarConditionProjector.scala [hunk 1]
```diff
@@ -209,7 +211,7 @@ class ColumnarConditionProjector(
             logInfo(s"has no next, return false")
             return false
           }
-
+          beforeEval = System.nanoTime()
```
Comment (xuechendi) [inline oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarConditionProjector.scala:214]: put beforeEval, some iterate may return at line 243, may be we should simply put beforeEval and afterEval at line 236 and start another beforeEval at line 254?

File: oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarGroupbyHashAggregation.scala [hunk 2]
```diff
@@ -178,7 +181,7 @@ class ColumnarGroupbyHashAggregation(
         if (resultColumnarBatch.numRows == 0) {
           resultColumnarBatch.close()
           val elapse = System.nanoTime - beforeAgg
```
Comment (xuechendi) [inline oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarGroupbyHashAggregation.scala:183]: beforeAgg is before cbIterator.next() at line 160, so last operator execution time will also be calucated

File: oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarShuffledHashJoin.scala [hunk 3]
```diff
@@ -154,6 +155,8 @@ class ColumnarShuffledHashJoin(
       probe_iterator.close()
       probe_iterator = null
     }
+    totalTime.merge(buildTime)
```
Comment (xuechendi) [inline oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarShuffledHashJoin.scala:158]: buildTime also contains buildIter.next()

Comment (zhouyuan) [reply oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarShuffledHashJoin.scala:158]: fixed

File: oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarConditionProjector.scala (hunk 1 above)
Comment (zhouyuan) [reply oap-native-sql/core/src/main/scala/com/intel/oap/expression/ColumnarConditionProjector.scala:215]: fixed

**Comment by zhouyuan** [comment]
@xuechendi thanks for the review!

**Pull request merged**